sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -942,10 +943,12 @@ export type JSONValue = boolean | number | string | null | JSONArray | JSONObjec
 interface Location_2 {
     // (undocumented)
     databaseAccountEndpoint: string;
+    // Warning: (ae-forgotten-export) The symbol "LocationUnavailability" needs to be exported by the entry point index.d.ts
+    //
```

Comment:
This warning indicates that the `index.ts` file for the package needs to export `LocationUnavailability` from the module where it is defined. This is required for the rolled-up types to function correctly.

Reply:
addressed